Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -0,0 +1,286 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
```

Comment:
Can we keep using `#if SENTRY_TARGET_REPLAY_SUPPORTED`?

Reply:
Swift doesn't have this Objc macros

Reply:
are you 100% sure? AFAIK simple macros work fine